contracts/LosslessSecurityOracle.sol

```diff
@@ -39,7 +39,10 @@ contract LosslessSecurityOracle is
         uint256 _subscription,
         IERC20 _subToken
     ) public initializer {
-        __Ownable_init();
+        __Context_init_unchained();
+        __Pausable_init_unchained();
+        __Ownable_init_unchained();
+        __AccessControl_init_unchained();
         _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
         setSubscriptionFee(_subscription);
```

Comment:
If we want to set 0 as a fee, it won't be possible because the function does not allow setting the same value meaning that constructor would fail. CVF-6